src/abi.js

```diff
@@ -268,8 +268,20 @@ module.exports = class ABI {
 
   static load(name, file) {
     let data = JSON.parse(fs.readFileSync(file))
-    return Array.isArray(data)
-      ? new ABI(name, file, immutable.fromJS(data))
-      : new ABI(name, file, immutable.fromJS(data.abi))
+
+    let abi = null
+    if (Array.isArray(data)) {
+      abi = data
+    } else if (data.abi !== undefined) {
+      abi = data.abi
+    } else if (data.compilerOutput !== undefined && data.compilerOutput.abi !== undefined) {
+      abi = data.compilerOutput.abi
+    }
+
+    if (abi === null || abi === undefined) {
+      throw Error(`Could not extract abi from ${file}`)
```

Comment:
Even though the field is called "abi" in JSON, let's spell it `ABI` for users because it's an intialism / acronym.

Reply:
you mentioned that on the first review 🤦‍♂️